portals/publisher/source/src/app/components/Apis/Create/AsyncAPI/ApiCreateAsyncAPI.jsx

```diff
@@ -349,6 +401,49 @@ export default function ApiCreateAsyncAPI(props) {
                                     ))}
                                 </TextField>
                             )}
+                            {!hideExternalEndpoint && (
+                                <>
+                                    <TextField
+                                        fullWidth
+                                        id='itest-id-api-external-endpoint-input'
+                                        label={(
+                                            <>
+                                                <FormattedMessage
+                                                    id='Apis.Create.AsyncAPI.ApiCreateAsyncAPI.externalEndpoint'
+                                                    defaultMessage='External Endpoint'
+                                                />
+                                                <sup className={classes.mandatoryStar}>*</sup>
+                                            </>
+                                        )}
+                                        name='externalEndpoint'
+                                        value={apiInputs.externalEndpoint}
+                                        onChange={handleOnChangeExternalEndpoint}
+                                        helperText={
+                                            !isValidExternalEndpoint && (
+                                                <div style={{ marginTop: '10px' }}>
+                                                    <FormattedMessage
+                                                        id={'Apis.Create.AsyncAPI.ApiCreateAsyncAPI'
+                                                        + '.externalEndpoint.error'}
+                                                        defaultMessage='Invalid Endpoint URL'
+                                                    />
+                                                </div>
+                                            )
+                                        }
+                                        error={!isValidExternalEndpoint}
+                                        margin='normal'
+                                        variant='outlined'
+                                    />
+                                    <MuiAlert severity='warning' className={classes.externalEndpointWarning}>
```

Comment:
Break the warning into title and description

Reply:
Done with https://github.com/wso2/apim-apps/pull/130/commits/dd728f986a1af819941d28f19887d9d997354a0f